package.json

```diff
@@ -17,34 +17,34 @@
   },
   "dependencies": {
     "@bcgov/smk": ">=1.0.7",
-    "@tmcw/togeojson": "^4.1.0",
-    "chalk": "^4.1.0",
-    "cors": "^2.8.5",
-    "csv-parse": "^4.12.0",
-    "express": "^4.17.1",
-    "fgdb": "^1.0.0",
-    "figlet": "^1.5.0",
+    "@tmcw/togeojson": "~4.5.0",
+    "chalk": "~4.1.0",
+    "cors": "2.8.5",
```

Comment:
Just curious why you are allowing patch versions to be updated for some packages and not others?

Reply:
> Just curious why you are allowing patch versions to be updated for some packages and not others?

For the procedure I used (https://github.com/bcgov/smk-cli/pull/92#issuecomment-1061244593) I was in general moving towards more restriction, so versions where a patch was specified were not changed except to remove "^"s. Are you suggesting we allow patch version updates for all versions, even where we didn't before?

Reply:
No, not quite. If we've previously targeted a specific version, we might want to leave that alone. ie. the only two dependencies that fall into this category would be glob and material-design-icons-iconfont.

I'm wondering why these package versions are prefaced with a tilde:
    "@tmcw/togeojson": "~4.5.0",
    "chalk": "~4.1.0",
    "csv-parse": "~4.16.0",
    "fgdb": "~1.0.0",
    "figlet": "~1.5.0",
    "open": "~7.4.0",
    "xmldom": "~0.6.0"

And these one's are not:
    "cors": "2.8.5",
    "express": "4.17.3",
    "inquirer": "7.3.3",
    "lodash": "4.17.21",
    "minimist": "1.2.5",
    "multer": "1.4.4",
    "node-fetch": "2.6.7",
    "proj4": "2.7.5",
    "semver": "7.3.5",
    "shapefile": "0.6.6",
    "shelljs": "0.8.5",
    "touch": "3.1.0",
    "unzipper": "0.10.11",
    "xml2js": "0.4.23",

Reply:
On further study, I think I misunderstood the effect of a nonzero patch value. I can't recall where I saw it, but I remember reading something that implied that ~1.2.0 would allow patch upgrades but ~1.2.3 would not. That's why I removed ^ in these cases and did not add ~.

However in https://github.com/npm/node-semver#versions, in the section on tilde ranges, It shows that ~1.2.3 would allow any patch version greater than equal to 3, but only allow a minor version of 2. 

I'll push a change adding ~ for versions that had ^ previously, here and in the smk review.